snuba/subscriptions/codecs.py

```diff
@@ -62,3 +69,44 @@ def encode(self, value: SubscriptionTaskResult) -> KafkaPayload:
             ).encode("utf-8"),
             [],
         )
+
+
+class SubscriptionScheduledTaskEncoder(Encoder[bytes, ScheduledTask[Subscription]]):
```

Comment:
`SubscriptionScheduledTaskCodec` ? Why doesn't this extend `Decoder` as well ?

Reply:
Oops, this was supposed to extend Codec` not `Encoder`. Thanks.